test/validators.js

```diff
@@ -8561,6 +8561,28 @@ describe('Validators', () => {
     });
   });
 
+  it('should validate strong passwords', () => {
+    test({
+      validator: 'isStrongPassword',
+      valid: [
+        '%2%k{7BsL"M%Kd6e',
+        'EXAMPLE of very long_password123!',
+        'mxH_+2vs&54_+H3P',
+        '+&DxJ=X7-4L8jRCD',
+        'etV*p%Nr6w&H%FeF',
```

Comment:
There is a need for tests that pass `returnScore as an option`

Reply:
Added the tests for this case in the `sanitizers.js` test file since with this option, the function essentially becomes a sanitizer, turning the string into a number. Hope that makes sense